xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -486,12 +503,18 @@
         <Parameter Name="count" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">The array used to store decompressed bytes.</param>
+        <param name="offset">The byte offset in <paramref name="buffer" /> at which the read bytes will be placed.</param>
+        <param name="count">The maximum number of decompressed bytes to read.</param>
+        <summary>Reads a number of decompressed bytes into the specified byte array.</summary>
+        <returns>The number of bytes that were decompressed into the byte array. If the end of the stream has been reached, zero or the number of bytes read is returned.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="buffer" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.InvalidOperationException">The <see cref="T:System.IO.Compression.CompressionMode" /> value was <see langword="Compress" /> when the object was created, or there is already an active asynchronous operation on this stream.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="offset" /> or <paramref name="count" /> is less than zero.</exception>
+        <exception cref="T:System.ArgumentException">The <paramref name="buffer" /> length minus the index starting point is less than <paramref name="count" />.</exception>
+        <exception cref="T:System.IO.InvalidDataException">The data is in an invalid format.</exception>
+        <exception cref="T:System.ObjectDisposedException">The underlying stream is null or closed.</exception>
       </Docs>
     </Member>
     <Member MemberName="ReadAsync">
```

Comment:
There are still a few APIs missing docs. Do you plan to cover those on the next update? e.g. Line 538

Reply:
Correct. I decided to start working with the APIs that had already been described in Stream, GZipStream and DeflateStream.
I will cover the rest of the BrotliStream, GZipStream and DeflateStream APIs in a future pull request. I'm currently investigating how they work.